core/shared/src/test/scala/laika/parse/code/languages/LanguageSpec.scala

```diff
@@ -642,6 +642,55 @@ class LanguageSpec extends AnyWordSpec with Matchers {
         keyword("PRIMARY"), space, keyword("KEY"), other(" ("), id("`BAZ_ID`"), other(")\n);")
       )
     }
+
+    "parse a YAML document" in {
+      val input =
+        """# Doc
+          |
+          |```yaml
+          |# Source: mychart/templates/configmaps.yaml
+          |apiVersion: v1
+          |kind: 'ConfigMap'
+          |metadata:
+          |  name: port_overriden # Some comment
+          |  tag: abc#def
+          |data:
+          |  host: "localhost"
+          |  port: 9876 # That's fine
+          |  timeout: 23.56
+          |  null_value: null
+          |  boolean: true
+          |a_sequence:
+          |  - Item 1
+          |  - 123
+          |  - key: value
+          |```
+        """.stripMargin
+
```

Comment:
It would be good to vary the syntax a bit with optional spaces before and after the colon to test the suggested changes.